beast/plotting/plot_noisemodel.py

```diff
@@ -13,8 +13,7 @@ def plot_noisemodel(
     noise_file_list,
     plot_file,
     samp=100,
-    color=["black", "red", "gold", "lime", "xkcd:azure"],
-    label=None,
+    cmap_name='jet',
```

Comment:
Maybe better to pick a different cmap as the default?  See https://matplotlib.org/stable/tutorials/colors/colormaps.html

Reply:
changed it to 'viridis'